SemanticColorizer/Constants.cs

```diff
@@ -17,5 +17,7 @@ public static class Constants
         public const String LocalFormat = "s.semantic-colorizer.local";
         public const String TypeSpecialFormat = "s.semantic-colorizer.type-special";
         public const String TypeNormalFormat = "s.semantic-colorizer.type-normal";
+        public const String TypeClassFormat = "class name";
+        public const String TypeStructFormat = "struct name";
```

Comment:
these string values are default formats? - I guess there could be a named constant somewhere else in Roslyn.